src/__tests__/Price.Stale.test.ts

```diff
@@ -0,0 +1,40 @@
+import { AccountInfo, clusterApiUrl, Connection, PublicKey } from '@solana/web3.js'
+import { Magic, MAX_SLOT_DIFFERENCE, parsePriceData, Version } from '../index'
+
+test('Product', (done) => {
```

Comment:
I think you can put this test in `Price.test.ts`. The files are allowed to contain multiple tests. Also the name of this test is wrong.

Reply:
Yes you're right. Moved into Price.test.ts.